python/paddle/fluid/tests/unittests/test_linalg_lstsq_op.py

```diff
@@ -0,0 +1,212 @@
+                    pass
+                else:
+                    raise RuntimeError("Check LSTSQ residuals static Failed")
+
+            if self.driver in ("gelsy", "gelsd", "gelss"):
+                if (np.abs(fetches[2] - self._output_rank) < 1e-6).any():
+                    pass
+                else:
+                    raise RuntimeError("Check LSTSQ rank static Failed")
+
+            if self.driver in ("gelsd", "gelss"):
+                if (np.abs(fetches[3] - self._output_sg_values) < 1e-6).any():
+                    pass
+                else:
+                    raise RuntimeError(
+                        "Check LSTSQ singular values static Failed")
+
+
+class LinalgLstsqTestCase(LinalgLstsqTestCase):
+    def init_config(self):
+        self.dtype = 'float64'
+        self.rcond = 1e-15
+        self.driver = "gels"
+        self._input_shape_1 = (5, 10)
+        self._input_shape_2 = (5, 5)
+
+
+class LinalgLstsqTestCaseRcond(LinalgLstsqTestCase):
+    def init_config(self):
+        self.dtype = 'float64'
+        self.rcond = 0.1
+        self.driver = "gels"
+        self._input_shape_1 = (3, 2)
+        self._input_shape_2 = (3, 3)
+
+
+class LinalgLstsqTestCaseGelsFloat32(LinalgLstsqTestCase):
+    def init_config(self):
+        self.dtype = 'float32'
+        self.rcond = 1e-15
+        self.driver = "gels"
+        self._input_shape_1 = (10, 5)
+        self._input_shape_2 = (10, 2)
+
+
+class LinalgLstsqTestCaseGelssFloat64(LinalgLstsqTestCase):
+    def init_config(self):
+        self.dtype = 'float64'
+        self.rcond = 1e-15
+        self.driver = "gelss"
+        self._input_shape_1 = (5, 5)
+        self._input_shape_2 = (5, 1)
+
+
+class LinalgLstsqTestCaseGelsyFloat32(LinalgLstsqTestCase):
+    def init_config(self):
+        self.dtype = 'float32'
+        self.rcond = 1e-15
+        self.driver = "gelsy"
+        self._input_shape_1 = (8, 2)
+        self._input_shape_2 = (8, 10)
+
+
+class LinalgLstsqTestCaseBatch1(LinalgLstsqTestCase):
+    def init_config(self):
+        self.dtype = 'float32'
+        self.rcond = 1e-15
+        self.driver = None
+        self._input_shape_1 = (2, 3, 10)
+        self._input_shape_2 = (2, 3, 4)
+
+
+class LinalgLstsqTestCaseBatch2(LinalgLstsqTestCase):
+    def init_config(self):
+        self.dtype = 'float64'
+        self.rcond = 1e-15
+        self.driver = "gelss"
+        self._input_shape_1 = (2, 8, 6)
+        self._input_shape_2 = (2, 8, 2)
+
+
+class LinalgLstsqTestCaseLarge1(LinalgLstsqTestCase):
+    def init_config(self):
+        self.dtype = 'float64'
+        self.rcond = 1e-15
+        self.driver = "gelsd"
+        self._input_shape_1 = (200, 100)
+        self._input_shape_2 = (200, 50)
+
+
+class LinalgLstsqTestCaseLarge2(LinalgLstsqTestCase):
+    def init_config(self):
+        self.dtype = 'float32'
+        self.rcond = 1e-15
+        self.driver = "gelss"
+        self._input_shape_1 = (50, 600)
+        self._input_shape_2 = (50, 300)
```

Comment:
The UT need to check the compute correctness

Reply:
The correctness of computation is checked in UT.